tests/std/tests/P0896R4_ranges_alg_binary_search/test.cpp

```diff
@@ -30,7 +31,7 @@ struct empty_ranges {
     template <ranges::forward_range Fwd>
     static constexpr void call() {
         // Validate empty ranges
-        const Fwd range{};
+        const Fwd range{std::span<P, 0>{}};

```

Comment:
Nitpick, this test is `using namespace std;`.

Reply:
Auditing.